open-sphere-base/core/src/main/java/io/opensphere/core/units/length/Length.java

```diff
@@ -412,6 +412,16 @@ public Length multiplyBy(double factor)
         return create(getClass(), getMagnitude() * factor);
     }
 
+    /**
+     * Returns a negated copy of this length.
+     *
+     * @return the negated length
+     */
+    public Length negate()
+    {
+        return create(getClass(), -getMagnitude());

```

Comment:
That feels like some funny synax there (the negated method call). I didn't know that actually worked.